sklearn/manifold/_mds.py

```diff
@@ -324,6 +401,11 @@ class MDS(BaseEstimator):
             Pre-computed dissimilarities are passed directly to ``fit`` and
             ``fit_transform``.
 
+    method: {'smacof', 'svd'}, default ='smacof'
```

Comment:
The numpydoc standard requires a space before that colon

Reply:
Do we prefer the name "solver" or "method" elsewhere in scikit-learn?

Reply:
You're right, I think solver.

Reply:
Done